### DISCUSSION TITLE
Support for Android TV

### DISCUSSION THREAD

**Discussion by erryKinq** [libre-tube/LibreTube#3617] [Ideas]
An option to use the app on Android TV would be great. It would be enough if the app supports landscape by default.

**Comment by MajorMorteza** [comment]
Yes, I tested it on my mi box, control support is useful but the ui does not support landscape.

**Comment by luckkmaxx** [comment]
I think for better experience is needed a tv ui. Better if can be switched in settings manually.

**Comment by SecularSteve** [comment]
I think it's better if the app had "Casting Integration" with the [SmartTube](https://github.com/yuliskov/SmartTubeNext) app.

The ability to Cast Videos from LibreTube to SmartTube out of the box without any manual configurations.

**Comment by pflavio** [comment]
Any updates in this issue? The app is great on mobile, but the current apk on Android TV is very barely usable. Basically its only the top left quarter of the screen and controls aren't really working (such as swipe down to refresh subscriptions). I understand its made for mobile, but a TV adoption would be amazing!
